Source/engine/animationinfo.cpp

```diff
@@ -98,6 +98,8 @@ void AnimationInfo::SetNewAnimation(std::optional<CelSprite> celSprite, int numb
 	RelevantFramesForDistributing = 0;
 	TickModifier = 0.0F;
 	IsPetrified = false;
+	triggerFrame = std::max(distributeFramesBeforeFrame - 1, 0);

```

Comment:
I am not happy about how distributeFramesBeforeFrame is named now with its new usage and meaning added.
Shall I rename it to something else?

Reply:
OK I screw this up. I need another argument pass here - because monsters actually do not load its anim the same like players... (considering trigger).

Reply:
Or I will just update trigger member value...